src/Misc.ino

```diff
@@ -1647,6 +1647,15 @@ String parseTemplate(String &tmpString, byte lineSize)
                                 case 'C':
                                   value = val == inverted ? "CLOSE" : " OPEN";
                                   break;
+                                case 'A':
+                                  value = val == inverted ? "AUTO" : " MAN";
+                                  break;
+                                case 'a':
+                                  value = val == inverted ? "A" : "M";
+                                  break;
+                                case 'H':
+                                  value = val == inverted ? "COLD" : " HOT";
+                                  break;
```

Comment:
There is a mix-up of what the letter for each case mean.
Sometimes it is the first option and sometimes the second.

Reply:
Hi,
yes I know. Some commands start with the same letter. I had to mix them up...